### ISSUE TITLE
enhance eslint-plugin-react jsx-props-no-spread-multi to report member expressions

### ISSUE THREAD

**Issue by camc314** [oxc-project/oxc#4894]
Currently, on `jsx-props-no-spread-multi` will report on the following code:
```tsx
<Test {...props} {...props} /> 
```

but it will not report on

```tsx
<Test {...props.foo} {...props.foo} /> 
```

we should enhance the rule to support this case
